src/providers/passport-bearer-auth.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  AuthenticationBindings,
+  AuthenticationStrategy,
+  UserProfileFactory,
+} from '@loopback/authentication';
+import {StrategyAdapter} from '@loopback/authentication-passport';
+import {inject, Provider} from '@loopback/core';
+import {Strategy, VerifyFunction} from 'passport-http-bearer';
+
+export const AUTH_STRATEGY_NAME = 'bearer';
+export class PassportBearerAuthProvider<GoodOracle>
```

Comment:
should probably contain `provider `in the filename to stay consistent in the naming convention